ledger/internal/eval_test.go

```diff
@@ -307,8 +309,201 @@ func TestPrivateTransactionGroup(t *testing.T) {
 	require.Error(t, err) // too many
 }
 
+func tealOpLogs(count int) []mocktracer.Event {
+	var log []mocktracer.Event
+
+	for i := 0; i < count; i++ {
+		log = append(log, mocktracer.BeforeOpcode(), mocktracer.AfterOpcode())
+	}
+
+	return log
+}
+
+func flatten(rows [][]mocktracer.Event) []mocktracer.Event {
+	var out []mocktracer.Event
+	for _, row := range rows {
+		out = append(out, row...)
+	}
+	return out
+}
+
+// byte 0x068101 is `#pragma version 6; int 1;`
```

Comment:
also move this down.

(makes me wonder if we're copying similar programs into a lot of different places)